site/en/tutorials/images/transfer_learning.ipynb

```diff
@@ -69,7 +69,7 @@
         "id": "hRTa3Ee15WsJ"
```

Comment:
<p>Try to avoid empty sections between headings.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/2081/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>OK, done!</p>